src/lib/Wst/Offchain/BuildTx/TransferLogic.hs

```diff
@@ -338,5 +350,9 @@ registerTransferScripts = case C.babbageBasedEra @era of
         hshSpending = C.hashScript $ C.PlutusScript C.plutusScriptVersion transferSpendingScript
         credSpending = C.StakeCredentialByScript hshSpending
 
+        hshSeizeSpending = C.hashScript $ C.PlutusScript C.plutusScriptVersion transferSeizeSpendingScript
+        credSeizeSpending = C.StakeCredentialByScript hshSeizeSpending
+
     addConwayStakeCredentialCertificate credMinting
     addConwayStakeCredentialCertificate credSpending
+    addConwayStakeCredentialCertificate credSeizeSpending
```

Comment:
Registration step for transfer script was missing issuer spend (seize) stake script